libraries/WiFi/src/WiFi.h

```diff
@@ -164,6 +165,9 @@ class WiFiClass {
     void set_ip_settings_ap();
     void set_ip_settings_sta(cy_wcm_connect_params_t *connect_params);
 
+    // timeout configuration
+    unsigned long _timeout = 10000; // 10 seconds
```

Comment:
As a general practice I like to add the units to the variable name. The variable on its own provides the info, the comment is not required, and easier to maintain:
`_timeout` -> `timeout_ms`